src/admin-bro-options.interface.ts

```diff
@@ -415,6 +420,25 @@ export type PageHandler = (
   context: PageContext,
 ) => Promise<any>
 
+/**
+ * Bundle options
+ *
+ * @alias BundlerOptions
+ * @memberof AdminBroOptions
+ * @example
+ * const adminBro = new AdminBro({
+    resources: [],
+    rootPath: '/admin',
+    babelConfig: './.adminbro.babelrc'
+   })
+ */
+export type BundlerOptions = {
+  /**
+   * The file path to babel config file or json object of babel config.
+   */
+  babelConfig?: Record<string, any> | string;
```

Comment:
doesn't babel provide typings for that? I think it would be better to use something like `BabelConfig | string`

Reply:
I have found the type definition from [@types/babel-core](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/babel-core).
And I replace Record<string, any> to that.
Thanks for the reminder 😄 .